pdc_client/__init__.py

```diff
@@ -120,12 +120,18 @@ class to perform requests.
             except KeyError:
                 print("'%s' must be specified in configuration file." % CONFIG_URL_KEY_NAME)
                 sys.exit(1)
-            ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME) if ssl_verify is None else ssl_verify
-            insecure = config.get(CONFIG_INSECURE_KEY_NAME)
-            if insecure is not None:
-                sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
-                    CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+            if ssl_verify is None:
+                ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME)
+                insecure = config.get(CONFIG_INSECURE_KEY_NAME)
```

Comment:
I think we should first give some warning for end user if ssl_verify and insecure has the same value, right?